apps/antalmanac/src/stores/Schedules.ts

```diff
@@ -50,10 +50,19 @@ export class Schedules {
         this.skeletonSchedules = [];
     }
 
+    getNextScheduleName(newScheduleName: string, scheduleIndex: number) {
+        const scheduleNames = this.getScheduleNames().filter((_, index) => index !== scheduleIndex);
```

Comment:
question: If I'm understanding this line correctly, we're getting all of the schedule names except the original. Can we splice to remove the given index instead?

Reply:
yeah; getScheduleNames() returns a deep copy, so it'll be fine. i'll change it to splice